source/examples/xgboost-dask-databricks/notebook.ipynb

```diff
@@ -0,0 +1,2163 @@
+{
```

Comment:
<p>We should mention destroyingt the Databricks cluster here again. And maybe also mention that if they just leave it the cluster will time out, which saves them money in case they forget.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>